src/conn/conn_api.c

```diff
@@ -1285,6 +1285,35 @@ __conn_rollback_to_stable(WT_CONNECTION *wt_conn, const char *config)
 err:	API_END_RET(session, ret);
 }
 
+/*
+ * __conn_calibrate_ticks --
+ *	Calibrate a ratio from rdtsc ticks to nanoseconds.
+ */
+static void
+__conn_calibrate_ticks(WT_SESSION_IMPL *session)
+{
+#if (defined __i386) || (defined __amd64)
```

Comment:
Does it make sense to add a HAVE_RDTSC instead of using this pair of flags?

Reply:
I took the defines from `__wt_rdtsc()`. That is a possibility. I'd like @fedorova's opinion. In particular, my investigation makes me wonder/think if we should be using `rdtscp` instead.

Reply:
@sueloverso, rdtscp prevents the hardware from reordering the rdtscp instruction with respect to other instructions, surrounding it. This would be important if we relied on capturing very fine-grained nanosecond-lasting events, but in our case we are interested in events that last milliseconds, or perhaps microseconds. So using rdtscp is not that important. There are two downsides for using rdtscp: (1) it has a much higher overhead in comparison to rdtsc, (2) it is available on fewer processors. That's why I chose to use rdtsc in the timing function.

Reply:
Thanks for those comments. I agree we do not need the precision of very fine-grained accuracy and higher overhead is definitely something we want to avoid. So I am good staying with rdtsc.

Reply:
I've created WT-3802 to track moving this to HAVE_RDTSC in WT-3802